bosh/jobs/cloud_controller_ng/spec

```diff
@@ -225,6 +225,14 @@ properties:
     default: 180
     description: "Maximum health check timeout (in seconds) that can be set for the app"
 
+  cc.shared_isolation_segment_name:
+    default: "shared"
+    description: |
+      Name of the shared isolation segment created at startup.
+      This field can be updated, but subject to the following caveat:
+         Using the name of an existing IS other than the default one will cause a deployment to fail.
+         To recover, redeploy using the old (current) name of the previous Isolation Segment.
```

Comment:
I'm confused about what this is saying. Does this mean "redeploy using the previous shared_isolation_segment_name"?